optimum/intel/openvino/utils.py

```diff
@@ -107,6 +111,26 @@
 }
 
 
+NEED_CONVERT_TO_FAST_TOKENIZER: List[type(PreTrainedTokenizer)] = [
+    CLIPTokenizer,
+]
+
+
+def maybe_convert_tokenizer_to_fast(
+    hf_tokenizer: PreTrainedTokenizer, tokenizer_path: Path
+) -> Union[PreTrainedTokenizer, PreTrainedTokenizerFast]:
+    if isinstance(hf_tokenizer, PreTrainedTokenizerFast):
+        return hf_tokenizer
+
+    if any(isinstance(hf_tokenizer, slow_class) for slow_class in NEED_CONVERT_TO_FAST_TOKENIZER):
```

Comment:
could be replaced with 
```suggestion
    if isinstance(hf_tokenizer, NEED_CONVERT_TO_FAST_TOKENIZER):
```
with `NEED_CONVERT_TO_FAST_TOKENIZER` a tuple

Reply:
Fixed.